### ISSUE TITLE
Local Host...  What's up with it?

### ISSUE THREAD

**Issue by hellachella** [hellachella/portfolio-main#3]
Hi  @ericadamski and @thomasjbradley

So I was happily plugging along with sitemap stuff and then all of the sudden, localhost will not display my internal pages (this happend juse after I added sitemap.xml).  Just my index shows up...  It seems to show up on Git..  I removed all of the sitemap stuff I had added, but localhost says it still can't find my pages.  

I have tried turning Jekyll off and then turning it back on again.  I have not turned off the whole terminal because I am afraid it will turn of ruby etc (PS, I don't know if turning off ruby is a thing..)

:|

M

**Comment by hellachella** [comment]
@ericadamski @thomasbradley

Nevermind, I got it back, but out of curiosity why would that happen?

**Comment by ericadamski** [comment]
Sometimes things get in an error state and resetting the entire thing is the only solution. From what your describing I am sure that was the issue.

**Comment by hellachella** [comment]
@ericadamski Yeah.. I am also having a problem with my image links.  Maybe you know what's up.  When I set my config file to have pretty permalinks, all of my images disappear.  Does this mean I would have to go in and put {{site.baseurl}}/ before every single image that I have?  Because that also wrecks my lightboxes.  What if I don't use pretty urls for my site and just let them read .html in the url instead of doing that to 32983742938472938472 image links and javascript?

Is there an easier fix?  I watched the videos about it all again looking for a solution to that problem..
